common/dynamicconfig/constants.go

```diff
@@ -48,170 +231,20 @@ const UnlimitedRPS = math.MaxInt32
 * So that our go-docs can have the full [documentation](https://pkg.go.dev/github.com/uber/cadence@v0.19.1/common/service/dynamicconfig#Key).
 ***/
 const (
-	UnknownKey Key = iota
+	UnknownIntKey IntKey = iota
 
 	// key for tests
-	TestGetPropertyKey
```

Comment:
this key completely disappeared, might mean mistakenly missing tests?

Reply:
it's an untyped key. deleted it because we're enforcing type